src/atopile/cli/build.py

```diff
@@ -94,21 +92,22 @@ def _init_python_app(build_ctx: BuildContext) -> Module:
         app_class = import_from_path(
             build_ctx.entry.file_path, build_ctx.entry.entry_section
         )
-    except (FileNotFoundError, ImportError) as e:
-        raise errors.UserPythonLoadError(
+    except Exception as e:
+        raise errors.UserPythonModuleError(
```

Comment:
I'd prefer a split with an explicit error for things like `FileNotFound` error because it gives people something better to go chase down